devise_2fa.gemspec

```diff
@@ -14,14 +15,18 @@ Gem::Specification.new do |gem|
   gem.license       = 'MIT'
 
   gem.files         = `git ls-files`.split($INPUT_RECORD_SEPARATOR)
-  gem.executables   = gem.files.grep(%r{^bin/}).map { |f| File.basename(f) }
   gem.test_files    = gem.files.grep(%r{^(test|spec|features)/})
   gem.require_paths = ['lib']
 
+  gem.files = Dir["{app,config,db,lib}/**/*", "MIT-LICENSE", "Rakefile", "README.md"]
+
+  gem.add_dependency "rails", "~> 5.0"
```

Comment:
I think that >= 4.2 should work here, maybe?

Reply:
I tried but there there were several 5.0 features

Reply:
Was it something we added or something that exists in lib?

Reply:
lib